src/test/java/de/mediathekview/mserver/crawler/srf/parser/SrfSendungOverviewJsonDeserializerTest.java

```diff
@@ -20,52 +20,52 @@
     return Arrays.asList(new Object[][] { 
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this array creation and simply pass the elements. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3878)